lib/sidekiq/throttled/strategy/concurrency.rb

```diff
@@ -42,7 +42,9 @@ def dynamic?
 
         # @return [Boolean] whenever job is throttled or not
         def throttled?(jid, *job_args)
-          return false unless (job_limit = limit(job_args))
+          job_limit = limit(job_args)
+          return false unless job_limit
+          return true if job_limit.zero?
```

Comment:
Oh. Good point! Love the idea.

Sidenote: what do you think about handling negative limits?

Reply:
Also, would be nice to have same for threshold as well.

Reply:
I can definitely add this to threshold as well. How would you like to handle negatives? Like below?
```ruby
return true if job_limit <= 0
```

Reply:
I was thinking more of:

``` ruby
return true unless job_limit.positive?
```